core/src/i18n/en.yaml

```
@@ -1378,7 +1378,7 @@ oauth2-clients:
     post-logout-redirect-uris: Post-Logout Redirect URIs
     redirect-uris: Redirect URIs
     response-types: Response Types
-    scope: Scope
+    scope: Scopes
```

I'd go with "scope", for example Luigi calls it like that https://docs.luigi-project.io/docs/authorization-configuration, even though there are multiple 
```suggestion
    scope: Scope
```

---

`scope defines the permissions to request at login.`
yeah, it's a single scope with a set of permissions.

---

But here you have scopes - https://docs.apigee.com/api-platform/security/oauth/working-scopes

---

Yup, and there's a `scope` in JSON
```
  "scope" : "A",
```
I guess they named the article like that, just like https://www.ocg.nsw.gov.au/child-safe-organisations/working-with-children-check, even though you could also work with one child.